prow/cluster/monitoring/additional-scrape-configs_secret.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: additional-scrape-configs
+  namespace: prow-monitoring
+stringData:
+  prometheus-additional.yaml: |
+    - job_name: k8s-prow-builds-boskos
+      metrics_path: /prometheus
```

Comment:
NIP: why isn't the default value `/metrics`  cool enough for `boskos`? :)

Reply:
/shrug